spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -81,6 +82,20 @@ def index
 
         expect(response).to redirect_to login_two_factor_piv_cac_path
       end
+
+      it 'redirects to phone when on mobile and user has phone' do
+        allow_any_instance_of(Browser).to receive(:mobile?).and_return(true)
+        user = create(:user, :with_phone)
+        stub_sign_in_before_2fa(user)
+        allow_any_instance_of(
+          TwoFactorAuthentication::PivCacPolicy,
+        ).to receive(:enabled?).and_return(true)
```

Comment:
could we factory the user so we don't have to stub?
```suggestion
        user = create(:user, :with_phone, :with_piv_or_cac)
        stub_sign_in_before_2fa(user)
```